src/components/cart/cart-item/cart-item.tsx

```diff
@@ -31,8 +31,18 @@ export const CartItem = ({ cartItem, className, isLast }: CartItemProps) => {
         }
     }
 
+    const isAvailable =
```

Comment:
is this handling variants availability?
I guess PARTIALLY_AVAILABLE comes in case when particular selected variant is in stock?

Reply:
From what I understand this is information about variant's availability... 'PARTIALLY_AVAILABLE' is still available to me :)